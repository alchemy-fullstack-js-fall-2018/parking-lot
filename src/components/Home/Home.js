import React from 'react';
import { Link } from 'react-router-dom';
import ROUTES from '../../routes';

const Home = () => {

  return (
    <div>
      <h2>Welcome!</h2>
      <p><Link to={ROUTES.ALL_CARS.linkTo()}>All Registered Cars</Link></p>
      <p><Link to={ROUTES.LOT_CARS.linkTo()}>All Cars in Lot</Link></p>
    </div>
  );
};

export default Home;
