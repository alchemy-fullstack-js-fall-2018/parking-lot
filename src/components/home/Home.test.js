import Home from './Home';
import React from 'react';
import { shallow } from 'enzyme';

describe('Home component', () => {

  it('renders a Home component', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
