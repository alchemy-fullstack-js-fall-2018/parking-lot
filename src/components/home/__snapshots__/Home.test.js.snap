// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders a Home component 1`] = `"<div id=\\"homePage\\"><h1>Welcome</h1><p>Chevron dick dastardly yosemite sam theodore roosevelt dodgy uncle clive, chevron hairy kid at school andrew weatherall dick dastardly will you do the fandango yosemite sam theodore roosevelt louis xiii dodgy uncle clive lorreto del mar brad pitt, lorreto del mar yosemite sam off-piste chevron walrus dick dastardly knavish rogue will you do the fandango andrew weatherall theodore roosevelt colonel mustard brad pitt dodgy uncle clive hairy kid at school goose louis xiii. Knavish rogue fox hunting wario? Cappuccino collector Daniel plainview el snort bad guy super mario, don’t panic cappuccino collector dick van dyke super mario dick van dyke timothy dalton bad guy el snort Daniel plainview cream bun disaster cream bun disaster, cardinal richelieu cappuccino collector Droopy dick van dyke el snort trimmed timothy dalton bad guy don’t panic Daniel plainview cream bun disaster grooming dick van dyke super mario cream bun disaster. Lip warmer consectetur basil fawlty dali mr frothy-top lorreto del mar godlike bogie basket, bogie basket bogie basket doctor strange lorreto del mar mr frothy-top godlike basil fawlty lip warmer dali professor plum consectetur david seaman,godlike nefarious mr frothy-top doctor strange lorreto del mar dali lando calrissian bogie baske</p></div>"`;
